### DISCUSSION TITLE
Check for updates built in

### DISCUSSION THREAD

**Discussion by IvanJuraj** [Alex313031/thorium#315] [General]
Hi I would like to ask is there any check for updates feature (for example like in brave)
Or are you working on it?
![image](https://github.com/Alex313031/thorium/assets/82110449/d2394aa9-e1f0-4b7a-97bb-064e8df1931f)

Thanks

**Comment by gz83** [comment]
Chromium does not have automatic updates by default, and Thorium inherits this.

In addition, if you are using the Windows version, there is a software that may help you update Thorium.

**Comment by IvanJuraj** [comment]
What kind of software?

**Reply by gz83** [reply]
https://codeberg.org/ltguillaume/thorium-winupdater

We're still working on making this software work better

**Comment by IvanJuraj** [comment]
Thanks <3

ne 22. 10. 2023 o 14:57 Ho Cheung ***@***.***> napísal(a):

**Comment by Kol9yN** [comment]
Its obvious thing for convenient usage of any product.
Please think how to make it works "out of the box".

**Comment by gz83** [comment]
Close old and inactive discussions.
